src/c4/charconv.hpp

```diff
@@ -963,6 +963,242 @@ inline size_t atou_first(csubstr str, T *v)
 #   pragma GCC diagnostic pop
 #endif
 
+//-----------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+namespace detail {
+template<class T> struct overflow_max {};
+template<> struct overflow_max<uint8_t>
+{
+    static csubstr value_dec() { return csubstr("255"); }
+    static bool is_oct_overflow(csubstr str) {
```

Comment:
I was going to say to break the opening bracket to the next line; but let's make this (and the similar methods below) one-liners. Less noise to look at.

Reply:
I thought they were already one-liners and missed it, should be ok now